package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -88,9 +89,17 @@ def parse(self, **kwargs):
             names = np.zeros(natoms, dtype=object)
             types = np.zeros(natoms, dtype=np.int)
             bonds = []
+            #Checks whether file contains periodic boundary box info 
```

Comment:
The comment would be better if it explains why this works. Currently, it only explains what the code does. So something along the lines of 


> PBC box information is stored optionally in tinker files at the beginning of a frame. The box is stored as (x, y, z, alpha, beta, gamma). If not box information is stored the second value (space separated) is an atom name. To detect if a box is stored we, therefore, need to check if the second value can be converted to float. This assumes there are no pure numerical atom names.


it is longer but much better understandable for a maintainer later/